numpy/core/src/multiarray/multiarray_tests.c.src

```diff
@@ -608,6 +609,13 @@ incref_elide_l(PyObject *dummy, PyObject *args)
     return res;
 }
 
+static PyObject *
+npy_char_deprecation(PyObject *dummy)
+{
```

Comment:
Need a note as to why this is done this way. I assume to get a deprecation warning.